components/BattleScreen/Menu/menu.module.css

```diff
@@ -0,0 +1,93 @@
+.menu {
```

Comment:
Can you safely delete this file now?

Reply:
Not yet, since I have not yet revised the menu, since it is also not actually used in this PR and also in this US. We deleted at the latest until the next US.